src/components/views/rooms/RoomList.js

```diff
@@ -269,24 +270,24 @@ module.exports = React.createClass({
     }, 500),
 
     // Update which rooms and users should appear in RoomList as dictated by selected tags
-    updateSelectedTagsRooms: function(dmRoomMap, updatedTags) {
+    updateSelectedTagsRoomsForGroups: function(dmRoomMap, tag) {
```

Comment:
this name doesn't really give me a clear idea of what it does. Can we do any better?